Formula/r/ryelang.rb

```diff
@@ -22,9 +22,11 @@ class Ryelang < Formula
 
   depends_on "go" => :build
 
+  conflicts_with "rye", because: "both install `rye` binaries"
+
   def install
     ENV["CGO_ENABLED"] = OS.mac? ? "1" : "0"
-    system "go", "build", *std_go_args(ldflags: "-s -w")
+    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"rye")
```

Comment:
probably need a symlink for backward compatibility?

Reply:
Done.